src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSinkUnitTest.java

```diff
@@ -64,9 +64,26 @@ public void readsSinkTest(String inputBam, String outputFile) throws IOException
         Assert.assertEquals(rddParallelReads.count(), rddParallelReads2.count());
     }
 
+    @Test(dataProvider = "loadReadsBAM", groups = "spark")
+    public void readsSinkShardedTest(String inputBam, String outputFile) throws IOException {
+        new File(outputFile).delete();
```

Comment:
`IOUtils.deleteRecursivelyOnExit` here too I think. We should probably add a `BaseTest.createTmpDir` that includes an automatic marking for delete at some point.